docs/faq.md

```diff
@@ -49,3 +49,36 @@ intermediates and outputs parallel with the `src` folder and not inside it,
 you can help optimize Git command performance for developers in the repository
 by limiting the number of files Git needs to consider for many common
 operations.
+
+### Why is Scalar a C# tool instead of built directly into Git?
+
+Scalar does a few things that Git currently has no precedent of doing and a few
+things that Git will never do.
+
+Git does not have a concept of running background jobs, which is critical to
+keeping our repositories running smoothly. Git does not have a concept of
+cache-servers, so we need to use the GVFS protocol during `scalar clone` to
+set up the cache-server URL. Scalar also creates the `.scalarCache` directory
+so multiple enlistments share objects from the remote, allowing a second
+`scalar clone` operation to be much faster than the first. All of these concepts
+are on our backlog for contributing to Git, in which case we can transition
+from the Scalar implementation to the Git implementation.
+
+Scalar uses the GVFS protocol for some repositories. This protocol was created
+by Azure Repos to support large repositories before the Git protocol supported
+partial clone and the Git client supported missing objects. In a way, the GVFS
+protocol proved that there was value in creating the partial clone feature.
+
+Scalar does currently require working with a
+[custom version of Git](https://github.com/microsoft/git), but we hope to relax
+that restriction eventually.
+We try to keep our fork of Git as close to [upstream](https://github.com/git-for-windows/git)
+as we can, but some features are necessary _for now_. Still, maintaining those

```

Comment:
Should we replace `features` with `patches` in this paragraph when talking about Git? A lot of the things in our fork are very surgical incisions, and not something on the order of big perf improvements or new features entirely.

Reply:
I was preparing a retort about how we have real features in our private fork (like `git-gvfs-helper`) but you are pointing out an important issue around _perception_ of our private fork. I'll use the patches term to show our intent: this fork only contains small things that we need for now and hope to remove.